source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,76 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.6 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: March 15, 2024**
+
+### Compatibility
+ - Updated minimum required Edge and Chrome versions to 120+.
+
+```{Important}
+If you upgrade from a release earlier than v9.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls version [v0.24.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.24.0).
+ - Pre-packaged GitLab plugin version [v1.8.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0).
+ - Added the [Outgoing OAuth Connections](https://mattermost.com/pl/outgoing-oauth-connections) integration type.
+ - Re-designed the **System Console > User Management** screen, and added the ability to batch export users in CSV format (Professional and Enterprise plans). On MySQL, users cannot view live results of the batch export in the user interface.
+ - Improved the appearance of profile/account menus.
+ - Added support for checkbox types in the **System Console** settings.
+ - Added support for WebP image previews in the web app similar to PNG and other image formats.
+ - Several pre-packaged plugins were removed.
+
+#### Administration
+ - Removed some unused Redux actions and reducers, including ``state.entities.posts.selectedPostId``.
```

Comment:
Wondering if any plugins are using this. I suspect not, but we should probably have an investigation/discussion on this sort of thing before executing on it. @hmhealey @M-ZubairAhmed Do you know what this reducer value is primarily used for, and if plugins may have a use case for using it?

Reply:
You actually asked about this above, but those actions haven't been used in quite a long time, if ever, in the web app. They carried over from the mobile app since it used mattermost-redux before the web app did